portfolio/src/main/webapp/script.js

```diff
@@ -94,18 +94,21 @@ function createCommentListItem(comment) {
 
   // Create the comment text to put into the list item
   const listText = document.createElement("span");
-  listText.innerText = comment.text;
+  listText.innerText = comment.nickname + ": " + comment.text;
+  listItem.appendChild(listText);
 
-  // Create the delete button to put into the list item
-  const deleteButton = document.createElement("button");
-  deleteButton.innerText = "Delete";
-  deleteButton.addEventListener("click", () => {
-    deleteComment(comment);
-    listItem.remove();
-  });
+  // Create the delete button to put into the list item if the logged in user
+  // owns the comment
```

Comment:
Add period. Additionally, instead of referencing specific code snippets like "list item," it may be clearer to simplify this comment to just explaining the purpose. For example, "Only show the delete button to the owner of the comment." or something similar might explain the purpose of this code more clearly.